### ISSUE TITLE
basePath not reflected in endpoint "operations" section

### ISSUE THREAD

**Issue by Caligatio** [swagger-api/swagger-ui#1124]
I currently have Swagger served up out of a subpath of my application and Swagger-UI works fine other than the fact that the "operations" section only reflects the Swagger Spec Path and not the full path.

Example:

```json
{"apiVersion":"0.1","swaggerVersion":"1.2","basePath":"http://localhost:3000/api","resourcePath":"/helloworld","apis":[{"path":"/helloworld","operations":[{"description":"simple GET example","summary":"returns {\"hello\":\"world\"}","paramters":[],"nickname":"helloworld","method":"GET","type":"void"}]}],"models":{}}
```

Swagger-UI correctly hits the endpoint at http://localhost:3000/api/helloworld but the page shows the [operation hitting "/helloworld"](https://github.com/swagger-api/swagger-ui/blob/master/src/main/template/operation.handlebars#L10) which is somewhat misleading (initial glance then makes it seem like the endpoint is actually http://localhost:3000/helloworld)

Would it be possible to still group endpoints with similar paths but to also tack on the relative basePath?

**Comment by fehguy** [comment]
Can you please confirm that this is using swagger-ui 2.1.0-M2?  And also, do you define a basePath at the Resource Listing (i.e. api-docs)?

**Comment by mohsen1** [comment]
Duplicate of #1139

**Comment by vlombardi** [comment]
I've seen the whole story of issue #1139, from its birth, until now.
It seems to be considered as "closed" but it will only be available in Milestone 2.1, not yet released at the moment.
So, I'd ask you, what should I do to make "Try it now!" link to use the basePath today?
Should I only wait or can I do some change by myself to make it work in my project? If this is the case, would someone tell me how to do it?

**Comment by marlonglopes** [comment]
If you are using  nginx  you need to set X-Forward-Proto header to https or
$scheme
Em 25/05/2015 13:37, "Vincenzo Lombardi" <notifications@github.com>
escreveu:

**Comment by vlombardi** [comment]
No nginx for me, but my post was referring to something else: issue 1139 and its solution

**Comment by vlombardi** [comment]
Issue #1139 on Swagger-ui is about the "Try it now!" link and it has been solved in milestone 2.1I was just only asking how to apply a patch/workaround before that milestone will be released in order to make it work. The problem is that nobody described the solution and I did not find that on GitHub...




     Il Lunedì 25 Maggio 2015 18:44, Marlon Gomes Lopes <notifications@github.com> ha scritto:
   

 If you are using nginx you need to set X-Forward-Proto header to https or
$scheme
Em 25/05/2015 13:37, "Vincenzo Lombardi" <notifications@github.com>
escreveu:

**Comment by webron** [comment]
@vlombardi - the issue is closed because it's been fixed (and another user commented that it was indeed fixed for them).

So, if you still have an issue, either comment on that issue (which was the focus issue) and ask to reopen, or open a new issue. However, since it has been reported to be fixed, I'd suggest first making sure you're using the latest version.
